lib/fauna/request_result.rb

```diff
@@ -11,8 +11,9 @@ class RequestResult
     # Request data.
     attr_reader :request_content
     ##
-    # Value returned by the response.
+    # Value returned by the server.
     # Includes "resource" wrapper hash, or may be an "errors" hash instead.
+    # May be nil if response_raw is not parseable json.
```

Comment:
`response_raw` is not part of `RequestResult`. Did you meant to include it here? Feels like it would be more straightforward that way.